axelor-account/src/main/resources/views/InvoiceLine.xml

```diff
@@ -187,15 +187,47 @@
         if="__config__.app.isApp('account') &amp;&amp; __config__.app.getApp('account').getManageAnalyticAccounting()"
         onTabSelect="action-invoice-line-attrs-read-only-distribution-lines">
         <field name="analyticDistributionTemplate"
-          readonlyIf="!account.analyticDistributionAuthorized"
+          if="__config__.app.isApp('account') &amp;&amp; __config__.app.getApp('account').getManageAnalyticAccounting()"
```

Comment:
I don't think `if="__config__.app.isApp('account')` is necessary as we are in `account` module
Same goes for other cases